src/main/java/org/czekalski/petapiintegration/apiClient/v1/dto/AnimalsListDto.java

```diff
@@ -0,0 +1,46 @@
+package org.czekalski.petapiintegration.apiClient.v1.dto;
+
+
+import com.fasterxml.jackson.annotation.*;
+
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class AnimalsListDto {
+    private final   List<AnimalDto> animals;
```

Comment:
Formatowanie
```suggestion
    private final List<AnimalDto> animals;
```

Reply:
Najlepiej puść na wszystkich tych plikach reformat :) Możesz też ustawić, żeby podczas commitowania kod był reformatowany. Czystość kodu jest ważna, bo wpływa na jego czytelność. Podczas rozmowy o pracę na pewno będą zwracali na to uwagę, ale na co dzień w pracy też :wink:

Reply:
Ok, ustawię następnym razem przy commicie